pkg/controller/certificaterequests/sync_test.go

```diff
@@ -168,7 +168,7 @@ func TestSync(t *testing.T) {
 				ExpectedActions:    []testpkg.Action{},
 			},
 		},
-		"should return nil (no action) if certificate request is denied": {
+		"should update RequestDenied if certificate request is denied": {
```

Comment:
"Should set the Ready condition status to reflect the denied status if the certificate request is denied"?